README.md

```diff
@@ -15,16 +15,7 @@ Velero consists of:
 * A server that runs on your cluster
 * A command-line client that runs locally
 
-You can run Velero in clusters on a cloud provider or on-premises. For detailed information, see [Compatible Storage Providers][99].
-
-## Installation
```

Comment:
What's the thinking for removing this section?

Reply:
To avoid having install instructions scarred in multiple places. The content under "More information" should be all that is necessary, and should be very high level pointing to one entry point, imo. But maybe we can make this better.

Reply:
OK, I guess since we now have `velero install` rather than static YAML this disclaimer isn't so important anymore. WFM.